src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -100,34 +78,20 @@
 helps['acr login'] = """
     type: command
     short-summary: Log in to a container registry through Docker.
-    examples:
-        - name: Log in to a container registry
-          text: >
-            az acr login -n MyRegistry
 """
 
 helps['acr show-usage'] = """
     type: command
     short-summary: Get the quota usages for a container registry.
-    examples:
-        - name: Get the quota usages for a container registry.
-          text: >
-            az acr show-usage -n MyRegistry
 """
 
 helps['acr credential show'] = """
     type: command
     short-summary: Get the login credentials for a container registry.
     examples:
-        - name: Get the login credentials for a container registry.
-          text: >
-            az acr credential show -n MyRegistry
-        - name: Get the username used to log into a container registry.
-          text: >
-            az acr credential show -n MyRegistry --query username
```

Comment:
This one could be useful as users may not know how to just get `username` using the query.

Reply:
Since we have a username/password example here, I think that might be sufficient, unless "username only" is a common case.